mediorum/server/cid_log_client.go

```diff
@@ -11,7 +11,7 @@ import (
 	"golang.org/x/exp/slog"
 )
 
-const CidLookupBatchSize = 2000
+const CidLookupBatchSize = 4000
```

Comment:
why increase?

Reply:
to get them to be able to serve all Qm CIDs sooner than like a full day or 2. some nodes I saw re-seed took a very long time to get all the Qm CIDs from large dbs like CN2. there's not as much load since this fetches from 1 node at a time, so I figure it's safer to bump now